js/cli/src/commands/configFile.ts

```diff
@@ -0,0 +1,40 @@
+import fs from "node:fs";
+
+import { dictionary, errorDebugString, number, object, string } from "idonttrustlikethat";
+
+export const readConfigFile = (): ConfigFile => {
+  const path = ".gatling/cli.json";
+  const content = fs.existsSync(path) ? fs.readFileSync(path, { encoding: "utf-8" }) : "{}";
+  const validated = configFileValidation.validate(JSON.parse(content));
+  if (validated.ok) {
+    return validated.value;
+  } else {
+    throw Error(`Gatling CLI configuration file is not valid.
+${errorDebugString(validated.errors)}`);
+  }
+};
+
+export type ConfigFile = typeof configFileValidation.T;
+
+const configFileValidation = object({
```

Comment:
Why would all these parameters except `memory`, `runParameters` and `controlPlaneUrl` be configurable by the end users? They shouldn't.